projects/plugins/crm/includes/ZeroBSCRM.Core.php

```diff
@@ -491,15 +491,6 @@ public static function instance() {
 		return self::$_instance;
 	}
 
-	/**
-	 * Cloning is forbidden.
-	 *
-	 * @since 2.1
-	 */
-	public function __clone() {
```

Comment:
We had to remove this to be able to clone the class in our base test case (this is to restore the global state of `$zbs` after each test to avoid mocking bleeding into other tests.